mysite/announcements/migrations/0007_remove_announcements_date_time_published.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-04 09:36
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('announcements', '0006_auto_20180604_1405'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='announcements',
+            name='date_time_published',
+        ),
+    ]
```

Comment:
Now, whenever we put up a Pull request for review. We should ensure that for a particular app we should have at most 1 database migration.

Why?
- While doing on local, we make many changes like adding a column, deleting a column, renaming a column, etc. before we arrive at a final solution. Now, but when we want to push something to production that is generally not the case, we don't want and need to make production DB to go through same steps, as there can be consequences.
- Running 1 migration is fast than running multiple.



So, here too we should club these migrations in 1.